crates/cli/src/templates/binding.java

```diff
@@ -0,0 +1,62 @@
+package PARSER_NS_CLEANED.jtreesitter.LOWER_PARSER_NAME;
+
+import java.lang.foreign.*;
+
+public final class PARSER_CLASS_NAME {
```

Comment:
Maybe add an explicit `private PARSER_CLASS_NAME() {}` constructor to override the implicit `public` default constructor? This avoids that users accidentally create instances of this class themselves.